geode-core/src/main/java/org/apache/geode/cache/client/internal/QueueStateImpl.java

```diff
@@ -310,7 +310,7 @@ void sendPeriodicAck() {
           success = true;
         } catch (Exception ex) {
           if (logger.isDebugEnabled()) {
-            logger.debug("Exception while sending an ack to the primary server: {}", ex);
+            logger.debug("Exception while sending an ack to the primary server:", ex);
```

Comment:
Is it possible that this production code intends to just log the message and not the entire stack trace?

Reply:
It's debug enabled, hence the entire stack is always better than just the message